GVRf/Framework/jni/objects/mesh.cpp

```diff
@@ -31,7 +31,7 @@
 #include "../gl/gl_program.h"
 
 namespace gvr {
-Mesh* Mesh::getBoundingBox() {
+Mesh* Mesh::createBoundingBox() {
 
```

Comment:
This call creates a new mesh object every time it is called, when in reality it only needs to be created when the bounding box is updated. We could have a cached copy of the mesh and invalidate it when the bounding box is updated. 

Let me know if you want me to fix it.